mmdet3d/ops/roiaware_pool3d/points_in_boxes.py

```diff
@@ -21,6 +21,13 @@ def points_in_boxes_gpu(points, boxes):
 
     box_idxs_of_pts = points.new_zeros((batch_size, num_points),
                                        dtype=torch.int).fill_(-1)
+
+    points_device = points.get_device()
+    assert points_device == boxes.get_device(), \
```

Comment:
Add comments to indicate why this is necessary. For example, add reference to issues